Sources/Views/Collections/CollectionIndex.swift

```diff
@@ -7,41 +7,184 @@ import PointFreeRouter
+          Class.size.width100pct,
+          Class.flex.flex,
+          Class.border.rounded.all,
+          Class.layout.overflowHidden,
+          Class.border.all,
+          Class.pf.colors.border.gray850,
         ])
       ],
-      .h3(
+      .a(
         attributes: [
-          .class([Class.h3])
+          .class([
+            Class.flex.flex,
+            Class.flex.justify.center,
+            Class.flex.align.center
+          ]),
+          .href(url(to: .collections(.show(collection.slug)))),
+          .style(unsafe: """
+            background: \(lower);
+            background: linear-gradient(\(index * 45)deg, \(lower) 0%, \(upper) 100%);
+            """)
         ],
-        "Collections"
+        .img(
+          base64: collectionsIconSvgBase64,
+          type: .image(.svg),
+          alt: "",
+          attributes: [
+            .style(margin(topBottom: .rem(5)))
+          ]
+        )
       ),
-      .ul(
-        .fragment(
-          Episode.Collection.all.map { collection in
-            .li(
-              .a(
-                attributes: [
-                  .href(url(to: .collections(.show(collection.slug))))
-                ],
-                .text(collection.title)
-              )
-            )
-          }
+      .div(
+        attributes: [
+          .class([
+            Class.padding([.mobile: [.top: 3, .leftRight: 3, .bottom: 2]])
+          ])
+        ],
+        .h6(
+          attributes: [
+            .class([
+              Class.pf.colors.fg.gray400,
+              Class.type.normal,
+              Class.pf.type.responsiveTitle8,
+              Class.margin([.mobile: [.all: 0]])
+            ])
+          ],
+          .text("Collection")
+        ),
+        .h4(
+          attributes: [
+            .class([
+              Class.pf.type.responsiveTitle4,
+              Class.type.normal,
+              Class.margin([.mobile: [.top: 0]])
+            ])
+          ],
+          .a(
+            attributes: [
+              .href(url(to: .collections(.show(collection.slug))))
+            ],
+            .text(collection.title)
+          )
+        ),
+        .p(
+          attributes: [
+            .class([
+              Class.padding([.mobile: [.all: 0]]),
+              Class.pf.type.body.regular,
+              Class.pf.colors.fg.black
+            ]),
+            .style(flex(grow: 1, shrink: 0, basis: .auto))
+          ],
+          .div(.markdownBlock(collection.blurb))
         )
+        // TODO: bring this back when we have time
+//        .a(
+//          attributes: [
+//            // TODO: figure out force unwrap
+//            .href(url(to: .collections(.show(collection.slug!)))),
+//            .class([
+//              Class.align.middle,
+//              Class.pf.colors.link.purple,
+//              Class.pf.type.body.regular,
+//              Class.margin([.mobile: [.top: 4]])
+//            ])
+//          ],
+//          .text("See collection (\(10) episodes)"),
```

Comment:
I wanna have a CTA in the collection card, but I was having trouble getting it to anchor to the bottom (even though I was able to do it for the pricing page :/). I come back to this later